src/utils.ts

```diff
@@ -480,21 +480,27 @@ const isLumigoStackTrace = (input) => {
   return LUMIGO_STACK_PATTERNS.some((word) => word.test(input));
 };
 
+export const removeLumigoFromError = (err) => {
+  const { stack } = err;
+    const stackArr = stack.split('\n');
+
+    const cleanedStack = stackArr.filter((v) => !isLumigoStackTrace(v));
+
+    err.stack = cleanedStack.join('\n');
+
+    return err;
+}
 export const removeLumigoFromStacktrace = (handleReturnValue) => {
   // Note: this function was copied to the auto-instrument-handler. Keep them both up to date.
   try {
     const { err, data, type } = handleReturnValue;
     if (!err || !err.stack) {
       return handleReturnValue;
     }
-    const { stack } = err;
-    const stackArr = stack.split('\n');
 
-    const cleanedStack = stackArr.filter((v) => !isLumigoStackTrace(v));
-
-    err.stack = cleanedStack.join('\n');
+    const sanitizedErr = removeLumigoFromError(err);
 
-    return { err, data, type };
+    return { sanitizedErr, data, type };
```

Comment:
This changes the signature of `removeLumigoFromStacktrace` from returning:
```js
{ err: err, data: data, type: type }
```
to:
```js
{ sanitizedErr: sanitizedErr, data: data, type: type }
```
so whoever deconstructs (unpacks) the value the old way:
```js
const { err, data, type } = removeLumigoFromStacktrace(...)
```
will get `err` as `undefined` - probably not what you meant (and it means that the current function is untested :( )

I think you better keep this the same by changing it to:
```suggestion
    return { err: sanitizedErr, data, type };
```

Reply:
Oh, I didn't realize that's how it works. Thanks for explaining. I'll change this.